### ISSUE TITLE
Two Forecasts For Same Location

### ISSUE THREAD

**Issue by paulp575** [vbguyny/ws4kp#128]
Why are there 2 different listings for the same location of Spokane International Airport, Spokane WA? What's the differences? It appears both have the same data - which does not agree with the official forecast?

**Comment by vbguyny** [comment]
They appear to be the same as they both return the same radar and zone ids.
The data used for the search bar comes from a previous implementation that was used for weather.gov years ago. I don't know the reason for this.

**Comment by paulp575** [comment]
Yes that is odd there's 2 different named locations for the same place.
And -
I'm using the (original) WS Simulator 4000 and their information is totally different than what your simulator uses. Hmmmm.
The original WS Simulator 4000 uses:
      State: WA
      NWSFO: OTX
      NWS Zone: 036
      FIPS Code: 063

Regardless, thanks for your efforts.
